ooniprobe/View/Settings/SettingsTableViewController.m

```diff
@@ -63,8 +65,9 @@ -(void)viewWillDisappear:(BOOL)animated{
         [testSuite.testList removeAllObjects];
         [testSuite getTestList];
     }
-    if (testSuite != nil || [[TestUtility getTestOptionTypes] containsObject:category])
+    if (testSuite != nil) {
```

Comment:
Same question as above: what is the impact of removing this clause?

Reply:
[TestUtility getTestOptionTypes] is no longer one of the expected settings type.